solutions/usaco-992.mdx

```diff
@@ -72,17 +76,20 @@ int main () {
 		int mid = lo + (hi - lo)/2;
 
 		if (ok(mid)) {
-			sol = max (sol, mid);
+			sol = max(sol, mid);
 			lo = mid + 1;
 		} else {
 			hi = mid - 1;
 		}
 	}
 
-	cout << (sol == top ? -1 : sol) << '\n';
+	cout << ((sol == top) ? -1 : sol) << '\n';
```

Comment:
why?

Reply:
clarity

Reply:
this line doesn't even look like it's necessary? can just print `sol`